AGPs/AGP-D.md

```diff
@@ -0,0 +1,32 @@
+---
+AGP: N/A
+Title: Aragon holiday
+Author: Luis Cuende (@luisivan)
+Status: Stage III
+Track: Proclamation
+Created: 2019-01-08
+---
+
+## Proclamation text
+
+The Aragon Network declares its national holiday on February 10th, called *Fight for freedom Day*.
+
+## Motivation for writing this proclamation
+
+The Aragon Network doesn't care about traditional nations and their local holidays. Yet, as a soon-to-be jurisdiction, having a holiday seems like a great way to set a date for the community to get together and celebrate.
+
+There are multiple dates that are relevant for the project. For example:
+
+- Nov 11th, 2016: The first idea for Aragon was born and Luis wrote the first whitepaper
+- Feb 10th, 2017: Luis and Jorge release Aragon to the world
+- May 17th, 2017: The Aragon Network token sale happened
+- Nov 17th, 2018: AGP-1 was ratified by ANT holders
+
+
+
+Out of those, I propose Feb 10th for the following reasons:
+
+- Releasing Aragon to the world was quite an achievement. There are lots of people with ideas, but not so many who can execute on them. On Feb 10th, anyone could create a decentralized organization with a few clicks. It was the beginning of a new era
+- It's the nearest date to the beginning of the year, which probably gathers more attention since usually there aren't a lot of local festivities during the first months of the year
+
+- It happens on Sunday during 2019 and on Monday during 2020
```

Comment:
I don't really see this as compelling... it's going to be on every day of the week eventually.

Reply:
I agree the day of the week it will be is a bit irrelevant, especially since it changes every year.